src/components/Navbar/index.css

```diff
@@ -1,6 +1,11 @@
 @import "../../styles/all.css";
 
 $modal-height: 402px;
+$navbar-background: color(white alpha(98%));
+$navbar-height: 68px;
+$navbar-height-mobile: 44px;
```

Comment:
And you should confirm this height with Inês

Reply:
These are browser side pixels, I measured manually.

Reply:
They depend on some values but I don't think it's productive to use `calc` on everything.

Reply:
As things are very much likely to change in the future.